pkg/build/package.go

```diff
@@ -653,6 +653,66 @@ func generatePkgConfigDeps(pc *PackageBuild, generated *config.Dependencies) err
 	return nil
 }
 
+// generatePythonDeps generates a python3~$VERSION dependency for packages which ship
+// Python modules.
+func generatePythonDeps(pc *PackageBuild, generated *config.Dependencies) error {
+	var pythonModuleVer string
+	pc.Logger.Printf("scanning for python modules...")
+
+	fsys := readlinkFS(pc.WorkspaceSubdir())
+	if err := fs.WalkDir(fsys, ".", func(path string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+
+		// Python modules are installed in paths such as /usr/lib/pythonX.Y/site-packages/...,
+		// so if we find a directory named site-packages, and its parent is a pythonX.Y directory,
```

Comment:
Should we look for `/usr/lib/pythonX.Y` specifically? Or would it make sense to find and process `/foo/bar/baz/pythonX.Y/site-packages` if we ever found it?

Either way this seems like a useful test case to document what paths are and aren't matched by this.

Reply:
There is presently no infrastructure to write test cases for the SCA engine.  I do not believe this PR should be blocked on that.

Reply:
> Or would it make sense to find and process /foo/bar/baz/pythonX.Y/site-packages if we ever found it?

Yes, it would.  Applications can adjust the module search path, but the pyX.Y component is always present.